registry/server/entries/entries.ts

```diff
@@ -0,0 +1,47 @@
+import express, { Request, Response } from 'express';
+import Joi from 'joi';
+import { EntryFactory } from '../common/services/entries/EntryFactory';
+import { IncorrectFqrnError } from '../common/services/entries/error/IncorrectFqrnError';
+import { NotFoundFqrnError } from '../common/services/entries/error/NotFoundFqrnError';
+import preProcessResponse from '../common/services/preProcessResponse';
+import { ValidationFqrnError } from '../common/services/entries/error/ValidationFqrnError';
+import { joiErrorToResponse } from '../util/helpers';
+import { AssetsManifestError } from '../common/services/assets/errors/AssetsManifestError';
+
+const EntriesRouter = express.Router();
+
+EntriesRouter.patch('/:fqrn', async (request: Request<{ fqrn: string }>, response: Response) => {
+    const fqrn = request.params.fqrn;
+    const params = request.body;
+    let entryService;
+
+    try {
+        entryService = EntryFactory.getFqrnInstance(fqrn);
+    } catch (error) {
+        if (error instanceof IncorrectFqrnError) {
+            return response.status(error.code).send(error.message);
+        }
+        throw error;
+    }
+
+    let results;
+    try {
+        results = await entryService.patch(params, { user: request.user });
+    } catch (error) {
+        if (error instanceof NotFoundFqrnError) {
```

Comment:
if (error instanceof NotFoundFqrnError || error instanceof ValidationFqrnError) {
    return response.status(error.code).send(error.message);
}

if (error instanceof Joi.ValidationError) {
    return response.status(422).send(joiErrorToResponse(error));
}
        
if (error instanceof AssetsManifestError) {
    return response.status(error.code).send(error.message);
}

Reply:
I think it make sence to optimize catching approach a bit later since all routes are supported single approach. 
In ideal case all errors should be presented in single format that could be parsed in the same manner.